events/syscall/redirect_stdout_stdin_to_network_connection_in_container.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2023 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"net"
+	"os"
+	"syscall"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(RedirectStdoutStdinFromContainer)
+
+var (
+	remoteAddr string = "localhost:8080"
+)
+
+func RedirectStdoutStdinFromContainer(h events.Helper) error {
+	if h.InContainer() {
+		listener, _ := net.Listen("tcp", remoteAddr)
+		defer listener.Close()
+		// Accept incoming connections in a separate goroutine
+		connChan := make(chan net.Conn)
+		go func() {
+			conn, _ := listener.Accept()
+			connChan <- conn
+		}()
+
+		// Create a client connection
+		clientConn, _ := net.Dial("tcp", remoteAddr)
+		defer clientConn.Close()
+
+		// Wait for the server connection
+		serverConn := <-connChan
+
+		// Redirect stdout to the network connection
+		redirectStdout(serverConn)
+	}
+	return &events.ErrSkipped{
+		Reason: "'Redirect Stdout/Stdin From Container' is applicable only to containers.",
+	}
+}
+
+func redirectStdout(conn net.Conn) error {
+	// Duplicate the file descriptor of the network connection
+	remoteFile, _ := conn.(*net.TCPConn).File()
+	defer remoteFile.Close()
+
+	// Duplicate the file descriptor of stdout
+	stdoutFile := os.Stdout.Fd()
+
+	// Redirect stdout to the network connection using dup2
+	if err := syscall.Dup2(int(remoteFile.Fd()), int(stdoutFile)); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
This seems to me a good candidate for `utils_linux.go` :thinking: @FedeDP wdyt?

Reply:
I agree @leogr !

Reply:
cc @h4l0gen

Reply:
@leogr changed done. PTAL again.